drivers/watchdog/Kconfig

```diff
@@ -104,4 +104,6 @@ source "drivers/watchdog/Kconfig.ti_tps382x"
 
 source "drivers/watchdog/Kconfig.tco"
 
+source "drivers/watchdog/Kconfig.xilinx"
```

Comment:
Please use `Kconfig.xlnx` as used in the rest of the tree.

Reply:
Changed.